recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -336,6 +337,8 @@ def export_sources(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if tools.Version(self.version) < "1.9":
+            delattr(self.options, "s3-crt")
```

Comment:
```suggestion
            del self.options.s3-crt
```

Reply:
I tried this piece of code before. It is not valid python because option name contains `-` character

Reply:
sorry, need to use underscore. It's automatically converted.

Reply:
It doesn't work. It is not automatically converted.
Running `conan create . 1.8.130@jcoulon/testing -o aws-sdk-cpp:text-to-speech=False -o aws-sdk-cpp:s3-crt=True`
- with my change, I get an early conan error `ERROR: aws-sdk-cpp/1.8.130@jcoulon/testing: option 's3-crt' doesn't exist`
- with your change, I get a later CMake error `s3-crt is required for build, but aws-cpp-sdk-s3-crt directory is missing!`

Reply:
I see, that's because the limitation using `-` with python ... Let's use your approach. I'll submit a hook validate this kind of situation.  Thank your for detailing!